### ISSUE TITLE
Ara plugin doesn't work on Ubuntu22.04

### ISSUE THREAD

**Issue by kemopq** [ansible-community/ara#487]
## What should be happening ?
It should work on Ubuntu22.04 the same as on Ubuntu20.04

**Comment by dmsimard** [comment]
Hey @kemopq and thanks for the issue.

At first glance, my understanding is that ansible was installed from Ubuntu's distribution packages (maybe from the Ubuntu Ansible PPA? I'm not sure) whereas ara would've been installed manually using pip.

If this is the case, it's possible you are running into issues caused by mixing python packages from the distribution (apt) and from pypi which can lead to a range of various different problems.

Could you confirm how you've installed both ara and ansible ?

Could you try reproducing the issue by installing both ansible and ara from PyPI ?
You can do this in a virtual environment like this:
```
python3 -m venv ~/virtualenv
source ~/virtualenv/bin/activate
pip install ansible ara
export ANSIBLE_CALLBACK_PLUGINS=$(python3 -m ara.setup.callback_plugins)
ansible-playbook <path/to/your/playbook.yml>
```

**Comment by kemopq** [comment]
You are right. The ansible is installed from Ubuntu Ansible ppa with apt as described in official ansible documentation. Using pip for installation of ansible solved the problem described above.
But still do you have any hint how to repair my original problem, as in our company official way to install ansible is by apt?
Thanks!

**Comment by dmsimard** [comment]
Hi,

I don't deploy to Ubuntu or Debian under your constraints so I don't have great suggestions but there is a debian package for ara maintained by the community though it isn't the latest version: https://tracker.debian.org/pkg/python-ara

I am not aware if there is a package for Ubuntu.
